flink-table/flink-table-planner/pom.xml

```diff
@@ -356,6 +356,7 @@ under the License.
 										<exclude>META-INF/*.DSA</exclude>
 										<exclude>META-INF/*.RSA</exclude>
 										<exclude>META-INF/services/java.sql.Driver</exclude>
+										<exclude>META-INF/versions/11/module-info.class</exclude>
```

Comment:
Can you explain this change?

Reply:
The new Jackson is available as a multi-release JAR, which is why there's a separate class for Java 11. The Flink shaded tests `test_table_shaded_dependencies.sh` checks if any .*class files exist to get a clean uber JAR. That's why I'm excluding it when creating the uber JAR.